### PR TITLE
bettercap 1.6.2 (new formula)

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/bettercap.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Bettercap < Formula
+  desc "Complete, modular, portable and easily extensible MITM framework"
+  homepage "https://www.bettercap.org/"
+  url "https://github.com/evilsocket/bettercap/archive/v1.6.2.tar.gz"
+  sha256 "1b364d7e31be5fa7b5f93eefe76763ad7bd4ac0b7b6bb4af05483157580a9cb9"
+
+  depends_on "openssl"
+  depends_on :ruby => "2.2.2"
```
Comment (dunn) [inline Formula/bettercap.rb]: I don't see this specified as a minimum version on the homepage; it just says 1.9 or greater.

**Comment by dunn** [comment]
The README says libpcap is a dep: https://github.com/evilsocket/bettercap#installation

Do we need to add that, or is the system version good enough?

File: Formula/bettercap.rb (hunk 1 above)
Comment (commitay) [reply Formula/bettercap.rb]: I think the minimum version is outdated.

```
ERROR:  Error installing bettercap:
	packetfu requires Ruby version >= 2.1.0.
```
```
ERROR:  Error installing bettercap:
	nio4r requires Ruby version >= 2.2.2.
```

**Comment by commitay** [comment]
System version seems to be good enough.

File: Formula/bettercap.rb (hunk 1 above)
Comment (dunn) [reply Formula/bettercap.rb]: Gotcha.  If you have a moment it would be nice to have that reported upstream.

**Comment by dunn** [comment]
@brewtestbot test this please

**Comment by dunn** [comment]
Thanks! Merged as 2e29fa3abb6dae383b75670e60f8fb38679ab71f.

**Comment by commitay** [comment]
Thanks @dunn!

**Pull request closed without merging**